CallAutomation_OutboundCalling/CallAutomation_OutboundCalling/Program.cs

```diff
@@ -111,7 +109,7 @@
     return Results.Ok();
 }).Produces(StatusCodes.Status200OK);
 
-app.MapPost("/filestatus", ([FromBody] EventGridEvent[] eventGridEvents, ILogger<Program> logger) =>
+app.MapPost("/api/recordingFileStatus", ([FromBody] EventGridEvent[] eventGridEvents, ILogger<Program> logger) =>
```

Comment:
nit: We don't need `[FromBody]` anymore as the framework will automatically figure out this is the payload model and will bind without issue.